### ISSUE TITLE
size mismatch error occured if training images and render images have different number.

### ISSUE THREAD

**Issue by Nplace-su** [OPEN-AIR-SUN/mars#108]
Hi，I met the same issue with #17 when I the number of training images and rendering frames are different. I trained a model with 250 images, and want to render 50 new view images, then I got the "size mismatch" error. Should the number of training images be any parameter of the model? Any chance we can avoid this? Thank you!

**Comment by wuzirui** [comment]
Yes, I think we should fix this. @xBeho1der

**Comment by xBeho1der** [comment]
Hi! @Nplace-su 

Would you like to provide more detailed information as I'm having trouble fully understanding your problems.

Are you looking to select a specific number of images from a pool of 250 images for rendering? If so, you can add some code here to select the images that you want to render.

![image](https://github.com/OPEN-AIR-SUN/mars/assets/107318439/ad742a2c-1c31-4fbe-bb68-a89248a22727)

 Or are you hoping to use the additional 50 images for rendering, in addition to the existing pool of 250 images?

**Comment by Nplace-su** [comment]
> Hi! @Nplace-su
> 
> Would you like to provide more detailed information as I'm having trouble fully understanding your problems.
> 
> Are you looking to select a specific number of images from a pool of 250 images for rendering? If so, you can add some code here in `scripts/cicai_render.py` to select the images that you want to render.
> 
> ![image](https://user-images.githubusercontent.com/107318439/285217355-ad742a2c-1c31-4fbe-bb68-a89248a22727.png)
> 
> Or are you hoping to use the additional 50 images for rendering, in addition to the existing pool of 250 images?

Oh, the original problem is that I trained a model using a sequence of 5 cameras(5 * 50 images), and I want to render a new view of one camera(50 images). I tried to edit the dataparser to read the camera poses sequence I want to render only, then I met the mismatch error. 
Your reply reminds me that I could still load the whole sequence and select part of them to render, thank you!

**Comment by xBeho1der** [comment]
The size mismatch error is raised while calling [load_state_dict](https://github.com/nerfstudio-project/nerfstudio/blob/5b4abc41e61805afcf849efa4289a27c72f29d01/nerfstudio/pipelines/base_pipeline.py#L109) because the size of `embedding_appearance` in nerfacto is related to the number of training images. However, I cannot give a definitive conclusion on the reasonableness of the number of training images as a model parameter. 😃

**Comment by amoghskanda** [comment]
Hey, I have trained CarNeRF on Scene05 using latents and state dict. The `first_frame` and `last_frame` values in `mars_kitti_dataparser.py` are 65 and 120 respectively. I want to render images from frame 65 till frame 210 for the same model. I changed the values in `cicai_render.py` but it gave me 
size mismatch for background_model.field.embedding_appearance.embedding.weight: copying a param with shape torch.Size([112, 32]) from checkpoint, the shape in current model is torch.Size([292, 32]).

Any workaround for this or should I retrain the model on images 65-210?
